client/src/app/site/site.component.ts

```diff
@@ -106,56 +75,7 @@ export class SiteComponent extends BaseComponent implements OnInit {
     public ngOnInit(): void {
         this.subscriptions.push(...this.getRouterSubscriptions());
         // observe the mainMenuService to receive toggle-requests
-        this.mainMenuService.toggleMenuSubject.subscribe((value: void) => this.toggleSideNav());
-
-        // get a translation via code: use the translation service
-        // this.translate.get('Motions').subscribe((res: string) => {
-        //      console.log('translation of motions in the target language: ' + res);
-        //  });
```

Comment:
I don't know, which purpose this code had. I deleted it, because it is only a comment.